src/ts_catalog/catalog.h

```diff
@@ -1004,9 +1004,16 @@ typedef enum Anum_continuous_aggs_bucket_function_pkey
 #define Natts_continuous_aggs_bucket_function_pkey                                                 \
 	(_Anum_continuous_aggs_bucket_function_pkey_max - 1)
 
-/****** CONTINUOUS_AGGS_HYPERTABLE_INVALIDATION_LOG_TABLE definitions*/
+/*
```

Comment:
I don't think these changes are needed any longer...

Reply:
This mention the location of the plugin name variable, which is the main modification.

Reply:
AFAICT, ` CONTINUOUS_AGGS_HYPERTABLE_INVALIDATION_SLOT_NAME` isn't used anywhere, for example.